src/blockencodings.cpp

```diff
@@ -101,9 +101,6 @@ ReadStatus PartiallyDownloadedBlock::InitData(const CBlockHeaderAndShortTxIDs& c
     const std::vector<std::pair<uint256, CTxMemPool::txiter>>& vTxHashes = pool->vTxHashes;
     for (size_t i = 0; i < vTxHashes.size(); i++) {
         uint64_t shortid = cmpctblock.GetShortID(vTxHashes[i].first);
-        //TODO: Put an ifdef guard around the prefetch here
-        if (i + 1 < vTxHashes.size())
```

Comment:
Thanks, if it's the result of a benchmark I'll believe you. I had assumed
that this was a leftover from when it required reading from various places
in memory.